### PR TITLE
Update to proper .NET sample format

### LINKED ISSUE #8715: Update samples to standard .NET format

### PR DESCRIPTION
**Pull request by annelo-msft**
Fixes https://github.com/Azure/azure-sdk-for-net/issues/8715

### CODE REVIEW

File: sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample2_AnalyzeSentiment.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    },
+    new TextDocumentInput("4", "Pike Place Market is my favorite Seattle attraction.  We had so much fun there.")
+    {
+         Language = "en",
+    }
+};
+
+AnalyzeSentimentResultCollection results = client.AnalyzeSentiment(inputs, new TextAnalyticsRequestOptions { IncludeStatistics = true });
+```
+
```
Comment (AlexanderSher) [inline sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample2_AnalyzeSentiment.md]: You can add similar header here ("To see the full example source files, see:")

File: sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample4_RecognizeEntities.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    }
+};
+
+RecognizeEntitiesResultCollection results = client.RecognizeEntities(inputs, new TextAnalyticsRequestOptions { IncludeStatistics = true });
+```
+
+* [Synchronously Sample4_RecognizeEntities.cs](../tests/samples/Sample4_RecognizeEntities.cs)
+* [Asynchronously Sample4_RecognizeEntitiesAsync.cs](../tests/samples/Sample4_RecognizeEntitiesAsync.cs)
+* [Synchronously Sample4_RecognizeEntitiesBatch.cs](../tests/samples/Sample4_RecognizeEntitiesBatch.cs)
+* [Synchronously Sample4_RecognizeEntitiesBatchConvienice.cs](../tests/samples/Sample4_RecognizeEntitiesBatchConvienice.cs)
```
Comment (AlexanderSher) [inline sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample4_RecognizeEntities.md]: This link is broken

Comment (heaths) [reply sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample4_RecognizeEntities.md]: FWIW, there's a nice extension for Code with id `blackmist.linkcheckmd` I've been using lately that is handy for link checks in markdown.

File: sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample5_RecognizePiiEntities.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+To recognize personally identifiable information in multiple text inputs as a batch, call `RecognizePiiEntities` on an `IEnumerable` of strings.  The results are returned as a `RecognizePiiEntitiesResultCollection`.
+
+```C# Snippet:TextAnalyticsSample5RecognizePiiEntitiesConvenience
+RecognizePiiEntitiesResultCollection results = client.RecognizePiiEntities(inputs);
+```
+
+To recognize personally identifiable information in a collection of text inputs in different languages, call `RecognizePiiEntities` on an `IEnumerable` of `TextDocumentInput` objects, setting the `Language` on each input.
+
+```C# Snippet:TextAnalyticsSample5RecognizePiiEntitiesBatch
+var inputs = new List<TextDocumentInput>
```
Comment (heaths) [inline sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample5_RecognizePiiEntities.md]: Are we using `var` in samples? Personally, I love 'em, but didn't think this was consistent.

**Review by heaths — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample2_AnalyzeSentiment.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+* [Sample2_AnalyzeSentiment.cs](../tests/samples/Sample2_AnalyzeSentiment.cs)
+* [Sample2_AnalyzeSentimentBatch.cs](../tests/samples/Sample2_AnalyzeSentimentBatch.cs)
+* [Sample2_AnalyzeSentimentBatchConvienice.cs](../tests/samples/Sample2_AnalyzeSentimentBatchConvienice.cs)
```
Comment (maririos) [inline sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample2_AnalyzeSentiment.md]:
```suggestion
* [Sample2_AnalyzeSentimentBatchConvenience.cs](../tests/samples/Sample2_AnalyzeSentimentBatchConvenience.cs)
```

File: sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample3_ExtractKeyPhrases.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+         Language = "en",
+    }
+};
+
+ExtractKeyPhrasesResultCollection results = client.ExtractKeyPhrases(inputs, new TextAnalyticsRequestOptions { IncludeStatistics = true });
+```
+
+* [Sample3_ExtractKeyPhrases.cs](../tests/samples/Sample3_ExtractKeyPhrases.cs)
+* [Sample3_ExtractKeyPhrasesBatch.cs](../tests/samples/Sample3_ExtractKeyPhrasesBatch.cs)
+* [Sample3_ExtractKeyPhrasesBatchConvienice.cs](../tests/samples/Sample3_ExtractKeyPhrasesBatchConvienice.cs)
```
Comment (maririos) [inline sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample3_ExtractKeyPhrases.md]:
```suggestion
* [Sample3_ExtractKeyPhrasesBatchConvenience.cs](../tests/samples/Sample3_ExtractKeyPhrasesBatchConvenience.cs)
```

**Review by maririos — APPROVED** [review]

**Pull request merged**